City-of-Helsinki/drupal-helfi-kymp
# UHF-2938: Add image paragraph to accordion item

How to test:

1. Checkout this branch.
2. On the root of the instance run `make drush-cr` `make drush-cim` `make drush-cr`
3. Go to edit any content that has accordion or add some. The accordion should now have image paragraph as option as well. Add some images to the paragraph and make sure the paragraph looks good.

Also check these PRs:
https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/210
https://github.com/City-of-Helsinki/drupal-hdbt/pull/186
https://github.com/City-of-Helsinki/drupal-helfi-strategia/pull/158
https://github.com/City-of-Helsinki/drupal-helfi-sote/pull/221
